modules/021-cni-cilium/images/builder/scripts/gops/build-gops.sh

```diff
@@ -20,9 +21,10 @@ git checkout -b "${gops_version}" "${gops_version}"
 git --no-pager remote -v
 git --no-pager log -1
 
-for arch in amd64; do
+for arch in amd64 arm64 ; do
```

Comment:
Do we need to build arm64 versions?

Reply:
we don't need build arm64, but it is original script from cilium and i think we should leave it untouched.